src/node_modules/react/lib/getNextDebugID.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
Forwarding modules is so much better than config aliases because it covers all tooling at once. E.g. webpack aliases are the worst because you have to replicate it in node, jest, flow, and anything else that need the module graph.